docs/release/release-tasks.md

```diff
@@ -332,7 +332,7 @@ The goal of this task to make the book for the current release available under e
     7. Write highlights section based on the initial release notes doc.
     8. Add the Kubernetes version support section and pay close attention to set the correct versions here.
        <br>**Note**: Check our [Kubernetes support policy](https://cluster-api.sigs.k8s.io/reference/versions.html#supported-kubernetes-versions) in the CAPI book. In case of doubt, reach out to the current release lead.
-    9. Modify `Changes since v1.x.y` to `Changes since v1.x`
+    9. **For minor releases** Modify `Changes since v1.x.y` to `Changes since v1.x`
```

Comment:
Just to check @sbueringer - do yo know if this was intentional?

Reply:
The previous version or the whole point?

I think overall we should end up with something that makes sense :D